Tests/fixtures/podcasting.yml

```diff
@@ -9069,3 +9069,335 @@
     model: ''
   os_family: iOS
   browser_family: Unknown
+-
+  user_agent: 'Podgrab - docker'
+  os: [ ]
+  client:
+    type: library
+    name: Podgrab
+    version: ""
+  device:
+    type: ""
+    brand: ''
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Provider 0.34-9e9009e7 for Radio Downloader 0.34-9e9009e7'
+  os: [ ]
+  client:
+    type: library
+    name: Radio Downloader
+    version: "0.34"
+  device:
+    type: ""
+    brand: ''
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'ESP32 HTTP Client/1.0'
+  os: [ ]
+  client:
+    type: library
+    name: ESP32 HTTP Client
+    version: "1.0"
+  device:
+    type: ""
+    brand: ''
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'com.google.android.apps.youtube.music/62354340 (Linux; U; Android 14; en_US; Pixel 7 Pro; Build/UP1A.231105.002; Cronet/114.0.5735.84)'
+  os:
+    name: Android
+    version: "14"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "62354340"
+  device:
+    type: smartphone
+    brand: Google
+    model: 'Pixel 7 Pro'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'YouTubeMusic/20231020.1.1 CFNetwork/1474 Darwin/23.0.0'
+  os:
+    name: iOS
+    version: "17.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "20231020.1.1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'YouTubeMusic/6.29.2 CFNetwork/1485 Darwin/23.1.0'
+  os:
+    name: iOS
+    version: "17.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "6.29.2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Dot;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
```

Comment:
Would it not be more like a library? https://github.com/alandtse/alexa_media_player

Reply:
If I am reading this right https://developer.amazon.com/en-US/docs/alexa/avs-device-sdk/mediaplayer.html and https://developer.amazon.com/en-US/docs/alexa/device-apis/alexa-media-playback.htmlit does not look like there ever was an official media player in Alexa, it's just that Alexa provided API to to have a media control (which explains the `tpapi` portion of the user agent). but even if we consider that this is not a library for home assistant, we do have a `mediaplayer` type.

Reply:
Interesting. Okay I guess I'll change it to Alexa, and change the client type to mediaplayer.  I never noticed it was mobile app. And I'll look at that home assistant project to ensure I'm not changing some user agent that it uses. If it is distinct, then I'll try and disambiguate it as a library client type, with some mention of home assistant.

Reply:
Updated.